src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerSubmissionControllerTests.cs

```diff
@@ -261,7 +261,7 @@ public async Task EditContactDetails_Post_ValidModel_ShouldRedirectToTaskList()
 
             // Assert
             result.Should().NotBeNull();
-            result.RouteValues["action"].Should().Be("TaskList");
+            result.RouteValues["action"].Should().Be("CheckAnswers");
```

Comment:
@stevelee-civica does this test pass now?  I'm not sure this change is needed

Reply:
restored the changes
it fails the test case now